patternlab/styleguide/source/assets/js/modules/expandClickable.js

```diff
@@ -0,0 +1,33 @@
+// export default (function (window, document) { // NEITHER OF THEM WORK WITH THIS.
```

Comment:
It looks like you would need to include jQuery to get this to work. You can change this line to something like this `export default (function (window, document, $) {` and then you would need to replace the bottom line with this `})(window, document, jQuery);`